doc/rbd/rbd-persistent-cache.rst

```diff
@@ -7,14 +7,14 @@
 Shared, Read-only Parent Image Cache
 ====================================
 
-`Cloned RBD images`_ from a parent usually only modify a small portion of
-the image. For example, in a VDI workload, the VMs are cloned from the same
-base image and initially only differ by hostname and IP address. During the
-booting stage, all of these VMs would re-read portions of duplicate parent
-image data from the RADOS cluster. If we have a local cache of the parent
-image, this will help to speed up the read process on one host, as well as
-to save the client to cluster network traffic.
-RBD shared read-only parent image cache requires explicitly enabling in
+`Cloned RBD images`_ usually only modify a small portion of the parent
+image. For example, in a VDI use-case, VMs are cloned from the same
+base image and initially differ only by hostname and IP address. During
+booting, all of these VMs read portions of the same parent
+image data. If we have a local cache of the parent
+image, this speeds up the read process on a given host.  We also realize
+reduction of client to cluster network traffic.
+RBD shared read-only parent image cache must be explicitly enabled in
```

Comment:
Is this "RBD-shared read-only parent-image cache"?

I'm genuinely asking out of rank ignorance.

Reply:
That's a lot of nouns used as adjectives that arguably don't add meaning, so I've tightened it up a bit.